nevergrad/optimization/optimizerlib.py

```diff
@@ -2031,7 +2032,8 @@ def __init__(self, parametrization: IntOrParameter, budget: tp.Optional[int] = N
             # pylint: disable=too-many-nested-blocks
             if self.has_noise and self.fully_continuous and self.dimension > 100:
                 # Waow, this is actually a discrete algorithm.
-                optimClass = ProgD13  # My guess is that we could do better by a sophisticated use of dimension / budget.
+                optimClass = ConfSplitOptimizer(num_optims=13, progressive=True,
```

Comment:
@teytaud why 13?

Reply:
13 because of experiments with 13 were the best...